Gulpfile.js

```diff
@@ -14,34 +19,39 @@ gulp.task('lint', function() {
 
 // Run the unit tests without any coverage calculations
 gulp.task('test', function() {
-  return gulp.src(['spec/**/*.js'])
+  return gulp.src([paths.spec])
       .pipe(jasmine());
 });
 
 // Task that calculates the unit test coverage for the module
 gulp.task('coverage', function() {
-  return gulp.src('lib/**/*.js')
-      .pipe(istanbul())
-      .pipe(istanbul.hookRequire())
-      .on('finish', function() {
-        gulp.src(['spec/**/*.js'])
-            .pipe(jasmine())
-            .pipe(istanbul.writeReports({
-              dir: 'build/coverage',
-              reportOpts: {dir: 'build/coverage'}
-            }));
-      });
+  return gulp.src(paths.lib)
+    .pipe(istanbul())
+    .pipe(istanbul.hookRequire())
+    .on('finish', function() {
+      gulp.src([paths.spec])
+        .pipe(jasmine())
+        .pipe(istanbul.writeReports({
+          dir: 'build/coverage',
+          reportOpts: {dir: 'build/coverage'}
+        }));
+    });
 });
 
 gulp.task('style', function() {
-  return gulp.src('lib/**/*.js')
+  return gulp.src(paths.lib)
     .pipe(jscs());
 });
 
 gulp.task('default', ['lint', 'style', 'test']);
 
-// On change to JavaScript files, run the default task
-gulp.task('dev', ['default'], function() {
-  gulp.watch(['spec/**/*.js', 'lib/**/*.js'], ['default']);
+// alias watch === dev
+gulp.task('watch', ['dev']);
+
+// On change to JavaScript files, run lint & test tasks
+// Do NOT run default: the style task breaks dev
+gulp.task('dev', ['lint', 'test'], function() {
+  gulp.watch([paths.spec, paths.lib], ['lint', 'test']);
```

Comment:
hmm the `style` task SHOULD work and not break `watch`, it works OK in other projects... Are you getting a specific error?